echopype/calibrate/calibrate_azfp.py

```diff
@@ -29,9 +29,15 @@ def get_cal_params(self, cal_params):
         cal_params : dict
         """
         # Params from the Beam group
```

Comment:
```suggestion
        # Params from Beam_group1 and Vendor group
```
or something similar

Reply:
Here I was trying to make it clear that some variables are grabbed from `Beam_group1` and others were from `Vendor`. I have added an additional comment and pushed this, so that is clear.

Reply:
Sounds good.